File: .config/hiss/testdata/HISS-04/python/negative/short-function.py
# SPDX-FileCopyrightText: 2026 lusoris <lusoris@pm.me>
#
# SPDX-License-Identifier: EUPL-1.2

"""A function inside every HISS-04 cap."""


def accumulate(values):
    """Short, single-branch, well inside the caps."""
    total = 0
    for value in values:
        total += value
    return total

File: .config/hiss/testdata/HISS-04/c/negative/short-function.c
// SPDX-FileCopyrightText: 2026 lusoris <lusoris@pm.me>
//
// SPDX-License-Identifier: EUPL-1.2

int accumulate(const int *xs, int n)
{
	int total = 0;
	for (int i = 0; i < n; i++)
		total += xs[i];
	return total;
}
